lib/rubocop/cop/rails/presence.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks code that can be written more easily using
+      # `Object#presence` defined by ActiveSupport.
+      #
+      # @example
+      #   # bad
+      #   a.present? ? a : nil
+      #
+      #   # bad
+      #   a.blank? ? nil : a
```

Comment:
I guess this should also cover negated versions of both methods. And there should probably a cop checking whether you can replace `!a.present?` with `a.blank?`. :-)

Reply:
> I guess this should also cover negated versions of both methods.

👍 

> And there should probably a cop checking whether you can replace !a.present? with a.blank?. :-)

Perhaps, `Rails/Present` cop or `Rails/Blank` cop checks them :)

Reply:
haha

At this point even I don't remember what we have cops for. 😆